codegen/src/test/resources/software/amazon/awssdk/codegen/poet/batchmanager/test-batchmanager-sync-interface.java

```diff
@@ -0,0 +1,118 @@
+package software.amazon.awssdk.services.batchmanagertest.batchmanager;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.concurrent.ScheduledExecutorService;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.core.batchmanager.BatchOverrideConfiguration;
+import software.amazon.awssdk.services.batchmanagertest.BatchManagerTestClient;
+import software.amazon.awssdk.services.batchmanagertest.batchmanager.internal.DefaultBatchManagerTestBatchManager;
+import software.amazon.awssdk.services.batchmanagertest.model.DeleteRequestRequest;
+import software.amazon.awssdk.services.batchmanagertest.model.DeleteRequestResponse;
+import software.amazon.awssdk.services.batchmanagertest.model.SendRequestRequest;
+import software.amazon.awssdk.services.batchmanagertest.model.SendRequestResponse;
+import software.amazon.awssdk.utils.SdkAutoCloseable;
+
+/**
+ * Batch manager class that implements automatic batching features for a BatchManagerTest sync client. This can be
+ * created using the static {@link #builder()} method.
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkPublicApi
+public interface BatchManagerTestBatchManager extends SdkAutoCloseable {
+    /**
+     * Buffers outgoing SendRequestResponses on the client and sends them as a SendRequestBatchRequest to SQS. Requests
+     * are batched together according to a batchKey and are sent periodically to BatchManagerTest. If the number of
```

Comment:
`batchKey` might be a bit vague

Reply:
Added javadoc to clarify what `batchKey` means.